src/charm.py

```diff
@@ -778,12 +832,37 @@ def _run_wp_cli(self, cmd, timeout=60, combine_stderr=False):
             group=self._WORDPRESS_GROUP,
             working_dir="/var/www/html",
             combine_stderr=combine_stderr,
-            timeout=timeout)
+            timeout=timeout
+        )
         try:
             stdout, stderr = process.wait_output()
-            return Result(0, stdout, stderr)
+            result = Result(0, stdout, stderr)
         except ops.pebble.ExecError as e:
-            return Result(e.exit_code, e.stdout, e.stderr)
+            result = Result(e.exit_code, e.stdout, e.stderr)
+        if combine_stderr:
+            logger.debug(f"Run wp-cli command: {cmd}\noutput: {result.stdout}")
+        else:
+            logger.debug(
+                f"Run wp-cli command: {cmd}\nstdout: {result.stdout}\nstderr:{result.stderr}"
```

Comment:
Please use `%s` formatting in log messages: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#formatting-log-messages

Reply:
I have updated this in a future PR.